### DISCUSSION TITLE
支持骁龙处理器吗？手里有个骁龙435的畅享

### DISCUSSION THREAD

**Discussion by AimerEGOIST** [kitsuned/PotatoNV#144] [Q&A]
感觉花钱读码又没必要，不读码吧，官方系统卡得一批

**Comment by bzy-080408** [comment]
并不支持，这个项目仅仅针对老旧麒麟CPU，如果不想读码可以试试找个工程包用9008把工程包的aboot刷进去，因为工程包的aboot通常是解锁状态的。但是工程包的aboot通常有很多问题。
